modules/sdk-coin-eth/src/lib/transactionBuilder.ts

```diff
@@ -748,4 +762,17 @@ export class TransactionBuilder extends BaseTransactionBuilder {
     }
     this._initCode = getProxyInitcode(implementationAddress);
   }
+
+  /**
+   * Set the wallet version for wallet to be initialized
+   *
+   * @param {number} version wallet version
+   */
+  walletVersion(version: number): void {
+    if (version < 0 || version > 2) {
```

Comment:
we support ETH wallets with version up to and including 3 https://github.com/BitGo/bitgo-microservices/blob/99011fce7f241688f32669674a19a6e12b7dbe4b/packages/api-docs/shared/CreateWallet.yaml#L61

Reply:
updating this. Thanks for point this out

Reply:
done